lib/webauto/web4cucumber.rb

```diff
@@ -137,20 +137,15 @@ def browser
       elsif @browser_type == :chrome
         logger.info "Launching Chrome"
 
-	      #https://bugs.chromium.org/p/chromium/issues/detail?id=1056073
-	      chrome_caps[:acceptInsecureCerts] = true
-        if Integer === @scroll_strategy
-          chrome_caps[:element_scroll_behavior] = @scroll_strategy
-        end
         if self.class.container?
           chrome_switches.concat %w[--no-sandbox --disable-setuid-sandbox --disable-gpu --disable-infobars --disable-dev-shm-usage]
         end
-        # options = Selenium::WebDriver::Chrome::Options.new
-        # options.add_extension proxy_chrome_ext_file if proxy_chrome_ext_file
-        options = {}
+        options = {browser_name: 'chrome',
+                   accept_insecure_certs: true}
+        options[:element_scroll_behavior] = @scroll_strategy if Integer === @scroll_strategy
         options[:extensions] = [proxy_chrome_ext_file] if proxy_chrome_ext_file
         url_or_switches = @selenium_url ? {url: @selenium_url} : {switches: chrome_switches}
-        @browser = Watir::Browser.new :chrome, desired_capabilities: chrome_caps, options: options, **url_or_switches
```

Comment:
`**url_or_switches` will carry either `url` or `switches` while `switches` is another deprecated param
```
12-07 09:15:45.193  2021-12-07 01:15:44 WARN Watir [DEPRECATION] ["switches_keyword", "capabilities"] :switches to initialize Browser is deprecated. Use :switches inside Hash with :options key instead; see explanation for this deprecation: http://watir.com/guides/capabilities.html.
```

Reply:
Nice catch, will take a look

Reply:
Pushed a commit and tested again. https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/245665/console

Reply:
@jhou1  it seems CI is still using old `watir` because we got different results when running locally and in CI